TeamCode/src/main/java/org/firstinspires/ftc/teamcode/helpers/commands/wrappers/setClawAngle.java

```diff
@@ -0,0 +1,13 @@
+package org.firstinspires.ftc.teamcode.helpers.commands.wrappers;
+
+import com.arcrobotics.ftclib.command.InstantCommand;
+
+import org.firstinspires.ftc.teamcode.subsystems.claw.ClawConfiguration;
+import org.firstinspires.ftc.teamcode.subsystems.claw.ClawSubsystem;
+
+public class setClawAngle extends InstantCommand {
+
+    public setClawAngle(ClawSubsystem claw, ClawConfiguration.TargetAngle angle){
```

Comment:
not gonna mark these cuz there is too much of them, but fix naming, remove the subsystem from the constructor AND    move the class files to subsystems.arm.commands

Reply:
and do the same for the claw